includes/admin.form.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @file
+ * Adminstration settings for the Islandora MARCXML module.
+ */
+
+/**
+ * Adminstration form for the Islandora MARCXML module.
+ */
+function islandora_marcxml_admin_settings(array $form, array &$form_state) {
+  $form['islandora_marcxml_allow_editing_of_existing_mods'] = array(
```

Comment:
@adam-vessey ups. Well, i did not test uninstalling it. Want to make  that a code tasks and attach a new pull request associated to the JIRA ticket?